apps/fabric-website-resources/src/customizations/customizations.ts

```diff
@@ -9,7 +10,8 @@ const exampleCardCustomizations: IExampleCardCustomizations[] = [
   { title: 'Word', customizations: WordCustomizations },
```

Comment:
Should we remove the fluent theme while we are at it? At least from the website.